deploy/cfm_deploy.yaml

```diff
@@ -98,16 +98,17 @@ Resources:
                 - "HEAD"
                 - "OPTIONS"
             TargetOriginId: "ExampleOrigin"
-            ForwardedValues:
-                QueryString: true
-                Cookies:
-                    Forward: "all"
+            OriginRequestPolicyConfig:
```

Comment:
What is the header policy here? 

Step 15 of https://docs.humansecurity.com/applications-and-accounts/docs/install-the-aws-lambdaedge-enforcer#upload-the-humanenforcer-lambda-required says to use the AllViewerAndCloudFrontHeaders-2022-06 for example -- is this included somewhere?

I think it's `allViewerAndWhitelistCloudFront`

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cloudfront-originrequestpolicy-headersconfig.html